packages/client/src/schema/tracing.ts

```diff
@@ -1,19 +1,32 @@
 export type AttributeDictionary = Record<string, string | number | boolean | undefined>;
 
+export type SetAttributesFn = (
```

Comment:
this one is unused it seems

Reply:
Yeah these are publicly exported types, so doesn't matter if we don't use them internally.